doc/devel/contribute.rst

```diff
@@ -161,6 +161,29 @@ please reach out on the :ref:`contributor_incubator`
 
 .. _`open an issue`: https://github.com/matplotlib/matplotlib/issues/new?assignees=&labels=Documentation&projects=&template=documentation.yml&title=%5BDoc%5D%3A+
 
+.. _contribute_triage:
+
+Triage
+------
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_ is our
+centralized location for feature requests, bug reports,
+tracking major projects and discussing priorities. It takes a lot of
+time and effort to keep oragnized, so curating our issue tracker is a valuable
+contribution. Some examples are:
```

Comment:
We can leave out the "time and effort" part. First, it doesn't sound very attractive to embark on something "taking a lot of time and effort", even if you don't have to do everything. Second, it sounds worse than it is. While it is work, I don't have the impression it's that much, e.g. compared to PR reviewing.

```suggestion
tracking major projects and discussing priorities. Helping to keep it
organized is a valuable contribution. Some examples are:
```

Alternative wording:

```suggestion
tracking major projects and discussing priorities. Curating it is a valuable
contribution. Some examples are:
```